blog/2025-08-00-uppy-5.0.mdx

```diff
@@ -0,0 +1,770 @@
+		></iframe>
+	</TabItem>
+	<TabItem value="vue" label="Vue">
+		<iframe
+			style={{ width: '100%', height: '500px' }}
+			src="https://stackblitz.com/github/transloadit/uppy/tree/main/examples/vue?file=package.json&embed=1&view=editor&showSidebar=1&hideTerminal=1"
+		></iframe>
+	</TabItem>
+	<TabItem value="svelte" label="Svelte">
+		<iframe
+			style={{ width: '100%', height: '500px' }}
+			src="https://stackblitz.com/github/transloadit/uppy/tree/main/examples/sveltekit?file=server.js&embed=1&view=editor&showSidebar=1&hideTerminal=1"
+		></iframe>
+	</TabItem>
+</Tabs>
+
+## Export maps for all packages
+
+All packages now have export maps. This is a breaking change in two cases:
+
+1. The css imports have changed from `@uppy[package]/dist/styles.min.css` to
+   `@uppy[package]/css/styles.min.css`
+2. You were importing something that wasn’t exported from the root, for instance
+   `@uppy/core/lib/foo.js`. You can now only import things we explicitly
+   exported.
+
+### Changed imports for React, Vue and Svelte
+
+Some components, like Dashboard, require a peer dependency to work but since all
+components were exported from a single file you were forced to install all peer
+dependencies. Even if you never imported, for instance, the status bar
+component.
+
+Every component that requires a peer dependency has now been moved to a subpath,
+such as `@uppy/react/dashboard`, so you only need to install the peer
+dependencies you need.
+
+Before:
+
+```javascript
+import { Dashboard } from '@uppy/react';
+```
+
+Now:
+
+```javascript
+import Dashboard from '@uppy/react/dashboard';
+```
+
+## Deprecations
+
+### Merged UI plugins
+
+`@uppy/status-bar` and `@uppy/informer` have been merged into `@uppy/dashboard`.
+These components have always been tightly coupled to `@uppy/dashboard` and with
+the advent of the new headless components and hooks, we want go all in those
+components and remove the confusing, inflexible ones.
+
+### Old plugins
+
+- `@uppy/store-redux` can be recreated by consumers, should they need it.
+- `@uppy/redux-dev-tools` is no longer needed.
+- `@uppy/progress-bar`, `@uppy/drag-drop`, `@uppy/file-input` and are superseded
+  by the new components and hooks.
+- `@uppy/aws-s3-multipart` has been merged in `@uppy/aws-s3` for over a year
+  now.
+
+## Overhauled tooling
+
+Just as the components started to age, so did our tooling. We put a lot of
+effect into migrating to a more modern monorepo setup.
+
+A fun example is comparing the
+[before](https://github.com/transloadit/uppy/blob/24fd4158a99aebdb50df8717c9a5d92707d16084/package.json)
+and [after](https://github.com/transloadit/uppy/blob/main/package.json) of our
+`package.json`.
+
+- Instead of global build scripts with lots of conditionals for packages, every
+  package now has its own build script.
+- Use TypeScript instead of Babel for compiling.
```

Comment:
shouldn't this be , Typescript Compiler instead of Typescript ? 

and transpiling instead of compiling ?